docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -239,6 +241,42 @@ Sentry.feedbackIntegration({
 });
 ```
 
+### Screenshot Button
+
+You can show a button that allows the user to take a screenshot in the Feedback Widget. The button is shown when `enableTakeScreenshot` is set to `true` in the `feedbackIntegration` method like in the example below.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
+Sentry.feedbackIntegration({
+  enableTakeScreenshot: true,
+  screenshotButtonOptions: {
+    styles: {
+      triggerButton: {
+        marginBottom: 75,
+      },
+    },
+  },
+})
```

Comment:
i: We could use a complete init so users don't misunderstand they can call feedbackIntegration outside of Sentry init


```suggestion
Sentry.init({
  integrations: [
    Sentry.feedbackIntegration({
      buttonOptions: {
        styles: {
          triggerButton: {
            marginBottom: 75,
          },
        },
      },
    }),
  ],
});
```